crates/sargon/src/profile/logic/profile_next_derivation.rs

```diff
@@ -86,6 +86,9 @@ impl Profile {
                             value.transaction_signing.factor_source_id
                                 == factor_source_id
                         }
+                        EntitySecurityState::Securified { value: _ } => {
+                            panic!("TODO check that") // TODO check that
+                        }
```

Comment:
TODO:
function that returns the next derivation

Reply:
Defer this, I've done this recently in https://github.com/radixdlt/sargon-mfa/pull/18